cli/cli_utils.go

```diff
@@ -325,6 +325,10 @@ func handleJWTSecret(flags *CliCmdFlags) error {
 		return fmt.Errorf(configs.GenerateJWTSecretError, err)
 	}
 
+	if err := os.MkdirAll(filepath.Dir(flags.jwtPath), os.ModePerm); err != nil {
```

Comment:
Change `os.ModePerm` for `0o755` in this function (the WriteFile was wrong all this time)

Reply:
Done cf622b0638997f38cfd9274a3331540624bdc38d